addons/workspace-dots/addon.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Editor workspace dots theme",
+  "description": "Allows the customisation of the style of the editor workspace dots.",
+  "tags": ["editor", "codeEditor", "theme"],
+  "versionAdded": "1.39.0",
+  "userscripts": [{ "url": "userscript.js", "matches": ["projects"] }],
+  "settings": [
+    {
+      "name": "Dots theme",
+      "id": "theme",
+      "type": "select",
+      "default": "dots",
+      "potentialValues": [
+        { "id": "dots", "name": "Default dots" },
+        { "id": "crosshairs", "name": "Crosshairs" },
+        { "id": "lines", "name": "Solid lines" },
+        { "id": "vertical", "name": "Vertical Lines" },
+        { "id": "horizontal", "name": "Horizontal Lines" },
+        { "id": "none", "name": "None" }
+      ]
+    },
+    {
+      "name": "Unnamed setting",
```

Comment:
This setting should be hidden if `theme` is `none`. I think "Spacing" would be a good name for it, but that would only make sense if higher values meant bigger spacing - the original spacing could be divided by 5 and then multiplied by this setting.

Reply:
> This setting should be hidden if `theme` is `none`.

Yes. I don't think "Spacing" is a good name thp